ansible/roles/ci/doc/best_practices.md

```diff
@@ -0,0 +1,92 @@
+# Best Practices
+
+## Splitting work between different CI servers
+
+Currently build tools support three hosted CI solutions 
+
+  * [Shipppable](https://www.shippable.com)
+  * [Semaphore](https://semaphoreci.com)
+  * [Circle](https://circleci.com)
+ 
+Also it supports local run which might be used by any CI setup on the local servers or for manual execution in Docker container.
+
+The one of the possible approaches is to split checks between different servers to make builds faster and to reduce cost for private repositories.
```

Comment:
"One of the possible approaches is to split checks between different servers to make builds faster and to reduce the cost for private repositories."